### PR TITLE
Update Test.java

### PR DESCRIPTION
**Pull request by yjshen324**

### CODE REVIEW

File: Test.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 public class Test {
 
   public static void main(String[] args) {
-    System.out.println("hello world");
+    System.out.println("hello robin");
```
Comment (yjshen324) [inline Test.java:4]:
```suggestion
    System.out.println("hello robot");
```

File: Test.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
 public class Test {
 
   public static void main(String[] args) {
-    System.out.println("hello world");
+    System.out.println("hello robot");
```
Comment (yjshen324) [inline Test.java:4]:
```suggestion
    System.out.println("hello robotics");
```

File: Test.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 public class Test {
 
   public static void main(String[] args) {
```
Comment (yjshen324) [inline Test.java:3]:
```suggestion
  public static void main(String args[]) {
```

**Review by yjshen324 — COMMENTED** [review]
great job. some suggestions though

File: Test.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
 public class Test {
```
Comment (yjshen324) [inline Test.java:1]:
```suggestion
public abstract class Test {
```

File: Test.java [hunk 5]
```diff
@@ -1,7 +1,7 @@
-public class Test {
+public abstract class Test {
```
Comment (robinshine) [inline Test.java:1]:
```suggestion
public abstract interface Test {
```

```suggestion
public interface Test {
```

File: Test.java [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
 
-  public static void main(String[] args) {
-    System.out.println("hello world");
+  public static void main(String args[]) {
+    System.out.println("hello shit");
```
Comment (robinshine) [inline Test.java:4]:
```suggestion
    System.out.println("hello man!");
```

**Comment by robinshine** [comment]
Somethings I am unhappy

**Review by robinshine — APPROVED** [review]
My 2 cents! please

**Pull request closed without merging**